sonar-dotnet-shared-library/pom.xml

```diff
@@ -14,7 +14,7 @@
   <description>Share code between C# and VB.NET plugins</description>
 
   <properties>
-    <sonar.version>6.7</sonar.version>
+    <sonar.version>7.3-alpha1</sonar.version>
```

Comment:
Let's also add the comment to say this is only the development API version.

Reply:
changed to 7.3